hack/install_clusterlink.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+set -e
+
+# Detrmine the OS.
+OS=$(uname)
+if [ "${OS}" = "Darwin" ] ; then
+  CL_OS="os"
+else
+  CL_OS="linux"
+fi
+
+
+# Detrmine the OS architecture.
+OS_ARCH=$(uname -m)
+case "${OS_ARCH}" in
+  x86_64|amd64)
+    CL_ARCH=amd64
+    ;;
+  armv8*|aarch64*|arm64)
+    ARCH=arm64
+    ;;
+  *)
+    echo "This ${OS_ARCH} architecture isn't supported"
+    exit 1
+    ;;
+esac
+
+filename="clusterlink-${CL_OS}-${CL_ARCH}.tar.gz"
+url="https://github.com/clusterlink-net/clusterlink/releases/download/${VERSION}/${filename}"
+
+# Set version to latest if not define and update the url.
+if [ "${VERSION}" = "" ] ; then
+  VERSION="latest"
+  url="https://github.com/clusterlink-net/clusterlink/releases/${VERSION}/download/${filename}"
+fi
+
+printf "\n Downloading %s from %s ...\n" "$filename" "$url"
+
+if ! curl -o /dev/null -sIf "$url"; then
+    printf "\n%s This version of clusterlonk wasn't found\n" "$url"
+    exit 1
+fi
+
+# Open the tar file.
```

Comment:
looking at [Go installation](https://go.dev/doc/install) (for example) they leave the steps of deciding where to install to (local or system path) to the user with a ~recommendation~ default to system.
Maybe we can follow the same and stop the script after the download stage (which, in the Go case, is also also in the docs, not in a script...).
Do we really need the script at all vs having everything in the documentation like Go?
[kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) also doesn't have a script, just download instructions.

Reply:
But you can check on your computer which programs use this method (pip, Jupyter..)
```shell
 ls ~/.local/bin/
``` 
I think that most of the users don't really care about it, and if someone does he can move it manually to any location he wants. The script prints in the end the location of the installation.

Reply:
I actually don't have a `~/.local/bin/` directory at all.
The above tools are python? Maybe this is a python specific convention?